### PR TITLE
Fixed Typo in LibreEHR_Directories.md

### PR DESCRIPTION
**Pull request by SoumyadeepJana**

### CODE REVIEW

**Comment by teryhill** [comment]
You need to do a gitnor on the sqlconf file. That is a local file and does not need to be changes in the main repo

File: sites/default/sqlconf.php [hunk 1]
```diff
@@ -1,16 +1,17 @@
 <?php
 //  LibreEHR
 //  MySQL Config
-//  Referenced from /library/sqlconf.php.
-
-global $disable_utf8_flag;
-$disable_utf8_flag = false;
```
Comment (teryhill) [inline sites/default/sqlconf.php]: @SoumyadeepJana  you will need to add these back and send a change

File: sites/default/sqlconf.php [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'libreehr';
-$pass	= 'libreehr';
-$dbase	= 'libreehr';
```
Comment (teryhill) [inline sites/default/sqlconf.php]: @SoumyadeepJana  you will need to add these back and send a change

File: sites/default/sqlconf.php [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
+$pass	= 'soumya228';
+$dbase	= 'libreehr1';
+
+//Added ability to disable
+//utf8 encoding - bm 05-2009
+global $disable_utf8_flag;
+$disable_utf8_flag = false;
```
Comment (teryhill) [inline sites/default/sqlconf.php]: remove these

File: sites/default/sqlconf.php [hunk 4]
```diff
@@ -19,12 +20,11 @@
 $sqlconf["login"] = $login;
 $sqlconf["pass"] = $pass;
 $sqlconf["dbase"] = $dbase;
-
 //////////////////////////
 //////////////////////////
 //////////////////////////
 //////DO NOT TOUCH THIS///
-$config = 0; /////////////
+$config = 1; /////////////
```
Comment (teryhill) [inline sites/default/sqlconf.php]: @SoumyadeepJana  you will need to change this back and send a change

**Comment by tmccormi** [comment]
It's best to work in a clone of the repo that is NOT the same as where you are running the web installation to avoid getting trash commits.

**Comment by teryhill** [comment]
Looks good merging

**Pull request merged**